nukkit/src/main/resources/config.yml

```diff
@@ -0,0 +1,2 @@
+api-key: ''
```

Comment:
You don't need to include a config.yml file, it is already there from the other versions of the plugin.